ocs_ci/ocs/constants.py

```diff
@@ -1948,8 +1948,8 @@
 ACM_4_7_MULTICLUSTER_URL = "/multicloud/infrastructure/clusters/managed"
 # Concatenated CA file for vcenter
 VSPHERE_CA_FILE_PATH = os.path.join(DATA_DIR, "vsphere_ca.crt")
-SSH_PRIV_KEY = os.path.expanduser(os.path.join(".ssh", "openshift-dev.pem"))
-SSH_PUB_KEY = os.path.expanduser(os.path.join(".ssh", "openshift-dev.pub"))
+SSH_PRIV_KEY = os.path.expanduser(os.path.join("~", ".ssh", "openshift-dev.pem"))
```

Comment:
I think we can rely on the config values we have here:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/default_config.yaml#L62-L63

Reply:
Updated in latest commit